dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -33,13 +33,16 @@ cache.config=Authentication Cache
 cache.config.desc=Controls the operation of the authentication cache.
 
 initialSize=Initial cache size
-initialSize.desc=Initial number of entries supported by the authentication cache.
+initialSize.desc=Initial number of entries supported by the local authentication cache. This setting does not apply to the JCache.
 
 maxSize=Maximum cache size
-maxSize.desc=Maximum number of entries supported by the authentication cache.
+maxSize.desc=Maximum number of entries supported by the local authentication cache. This setting does not apply to the JCache.
 
 timeout=Cache eviction timeout
-timeout.desc=Amount of time after which an entry in the cache will be removed.
+timeout.desc=Amount of time after which an entry in the local authentication cache will be removed. This setting does not apply to the JCache.
```

Comment:
The duration until an entry in the local authentication cache is removed. This setting does not apply to the JCache cache.

Reply:
Updated. Also changed "local" to "in-memory".